pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -47,6 +48,8 @@ const (
 	scaleUnknown = -1
 	probePeriod  = 1 * time.Second
 	probeTimeout = 45 * time.Second
+	probePath    = "/_internal/knative/autoscaler/probe"
```

Comment:
So we don't probe autoscaler. Or is it the source?

Reply:
It's the source. Currently, when activator probes, it uses the probePath of "/_internal/knative/autoscaler/probe". Do you think different paths are required, or can we just have a whitelisted common path that is probed from all components?

Reply:
just making sure.